### PR TITLE
MathFu 1.0.2

### PR DESCRIPTION
**Pull request by AdrianArroyoCalle**
https://codein.withgoogle.com/dashboard/task-instances/6331223177691136/?sp-page=1

MathFu is a header only library. CMake is used only to build unit tests (but it uses Git submodules for that also).

### CODE REVIEW

File: dev-games/mathfu/mathfu-1.0.2.recipe [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:cmake
+	cmd:git
```
Comment (waddlesplash) [inline dev-games/mathfu/mathfu-1.0.2.recipe]: Is this really needed?

**Comment by waddlesplash** [comment]
You don't appear to use CMake in the recipe...?

**Comment by AdrianArroyoCalle** [comment]
You're right. If it's a header only library it can't require CMake or GCC...

**Comment by waddlesplash** [comment]
Thanks!

**Pull request merged**